src/textual_summary/stories/GenericGroup.stories.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import GenericGroup from '../generic_group';
+import { genericGroupData } from '../data/generic_group';
+
+storiesOf('TextualSummary', module)
+  .add('GenericGroup', () => {
+    return (
+      <GenericGroup items={genericGroupData.items} title={genericGroupData.title} onClick={e => null} />
```

Comment:
What is the purpose of `onClick={e => null}`? I would advise specifying this inside the `GenericGroup` component and simply don't pass any props.

You can use defaultProps of that component. Something like
```
GenericGroup.propTypes = {
  onClick: PropTypes.func
...
}

GenericGroup.defaultProps = {
  onClick: () => {}
}
```

Reply:
For a real use the `onClick` is mandatory. In the Storybook, I don't care.

I can replace it with some `alert("You clicked!")` if you want. But it looks silly.

No I don't want to have a default value. For a real use there needs to be a handler.

See https://github.com/ManageIQ/manageiq-ui-classic/pull/3420/files#diff-ea54a250fdb683370b39733f087cfbdeR5

There I have a default on-click handler. BUT: that behavior is manageiq-ui-classic specific internal. For that reason it's there and not here.

Does that address your concerns?

Reply:
Ahh ok i thought that there is some rule. Never mind then

Reply:
I don't like `e => null` as well, since it does not give any information to user of such storybook. Can you please use https://github.com/storybooks/storybook/tree/master/addons/actions so the action is logged at least into storybook action list?